tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/wizard/template/CDIWebProjectWizardTemplate.java

```diff
@@ -104,25 +69,29 @@ public void createCDIProjectWithoutBeansXml(){
 		CDIProjectWizard cw = new CDIProjectWizard();
 		cw.open();
 		WebProjectFirstPage fp = new WebProjectFirstPage(cw);
-		fp.setProjectName(PROJECT_NAME);
+		fp.setProjectName(PROJECT_NAME + "2");
```

Comment:
do really need it adding +2 to project name? if so, please use local variable in this method

Reply:
Well sorry for that. Fixed.